src/esp/metadata/attributes/LightLayoutAttributes.h

```diff
@@ -143,23 +143,20 @@ class LightInstanceAttributes : public AbstractAttributes {
    * use that to build this data.
    */
   std::string getObjectInfoInternal() const override {
-    return cfg.value("position")
+    return getAsString("position")
         .append(1, ',')
-        .append(cfg.value("direction"))
+        .append(getAsString("direction"))
         .append(1, ',')
-        .append(cfg.value("color"))
+        .append(getAsString("color"))
         .append(1, ',')
-        .append(cfg.value("intensity"))
+        .append(std::to_string(getIntensity()))
         .append(1, ',')
         .append(getCurrLightTypeName())
         .append(1, ',')
         .append(getCurrLightPositionModelName())
         .append(1, ',');
```

Comment:
Ow, the allocations... Use [`Utility::formatString()`](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility.html#a5b85466788fc3d71b143909dda1ef44a), please :)

```cpp
return Cr::Utility::formatString("{},{},{},{},{},{}", getAsString("position"), getAs...);
```

For even less temporary string allocations format the Vector3 components directly, instead of taking them as a string separately.

Reply:
Thanks! forgot about that!

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.